static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -291,6 +293,21 @@ export function getOnboardingTasks({
       actionType: 'app',
       location: `/organizations/${organization.slug}/projects/new/`,
       display: true,
+      SupplementComponent: withApi(({task}: FirstEventWaiterProps) => {
+        if (hasQuickStartUpdatesFeature(organization)) {
+          if (!projects?.length || task.requisiteTasks.length > 0 || taskIsDone(task)) {
+            return null;
+          }
+          return (
+            <EventWaitingIndicator
+              text={t('Waiting for error')}
+              hasQuickStartUpdatesFeature
+            />
+          );
```

Comment:
his task is only considered complete when the user sends an error event. Therefore, we should utilize the signalizer in the same way we do when waiting for the first error.